fuzzing/fuzzer.go

```diff
@@ -787,13 +802,17 @@ func (f *Fuzzer) printMetricsLoop() {
 		memoryUsedMB := memStats.Alloc / 1024 / 1024
 		memoryTotalMB := memStats.Sys / 1024 / 1024
 
+		// Calculate total tests
+		totalTests := f.numTestsPassed + f.numTestsFailed
+
 		// Print a metrics update
 		logBuffer := logging.NewLogBuffer()
 		logBuffer.Append(colors.Bold, "fuzz: ", colors.Reset)
 		logBuffer.Append("elapsed: ", colors.Bold, time.Since(startTime).Round(time.Second).String(), colors.Reset)
 		logBuffer.Append(", calls: ", colors.Bold, fmt.Sprintf("%d (%d/sec)", callsTested, uint64(float64(new(big.Int).Sub(callsTested, lastCallsTested).Uint64())/secondsSinceLastUpdate)), colors.Reset)
 		logBuffer.Append(", seq/s: ", colors.Bold, fmt.Sprintf("%d", uint64(float64(new(big.Int).Sub(sequencesTested, lastSequencesTested).Uint64())/secondsSinceLastUpdate)), colors.Reset)
 		logBuffer.Append(", coverage: ", colors.Bold, fmt.Sprintf("%d", f.corpus.ActiveMutableSequenceCount()), colors.Reset)
+		logBuffer.Append(", failed: ", colors.Bold, fmt.Sprintf("%d/%d", f.numTestsFailed, totalTests), colors.Reset)
```

Comment:
I don't think it's necessary to track the `numTestsPassed` and sum the total. Instead `sequencesTested` can be used as the total.